cirq/google/api/v2/sweeps_test.py

```diff
@@ -18,7 +18,7 @@
 import sympy
 
 import cirq
-from cirq.api.google.v2 import run_context_pb2
+from cirq.google.api.v2 import run_context_pb2
```

Comment:
Could remove this and refer to `v2.run_context_pb2` below.

Reply:
Done.